src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -87,4 +87,8 @@ private static double clampPower(double power) {
     public void stop() {
         m_driveTrain.stopMotor();
     }
+
+    public double getMotorSpeed() {
+        return m_leftA.get();
+    }
```

Comment:
Maybe this should return both motor speeds (in an array for example). Otherwise we're only ever able to log the speed of one motor. Alternatively you could create a `getLeftMotorSpeed` and `getRightMotorSpeed`.